File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include<math.h>



int main(void){
    
    float dollars;
    do{
        dollars = get_float("Change owed: ");
    }
    while(dollars < 0);
    
    int cents = round(dollars * 100);

    int count = 0;
    
    do{
        if( cents >= 25 ){
        cents -= 25;
        count++;
        }
        
        else if(cents >= 10){
            cents -= 10;
            count++;
        }
        
        else if(cents >= 5){
            cents -= 5;
            count++;
        }
        
        else if(cents >= 1){
            cents -= 10;
            count++;
        }
    }
    while(cents > 0);
    
    printf("%i\n",count);
    
    
}

File: pset6/cash/cash.py
from cs50 import get_float


dollars = get_float("Enter postive number: ")

# to round cents 
cents = round(dollars * 100)

# intial value for counters
count = 0
quarters = 0
dimes = 0
nickels = 0 
pennies = 0 

# loop to count cions
while cents != 0:
    if cents >= 25:
        cents -= 25
        count += 1
        quarters += 1
        
    elif cents >= 10:
        cents -= 10
        count += 1
        dimes += 1
        
    elif cents >= 5:
        cents -= 5
        count += 1
        nickels += 1
        
    elif cents >= 1:
        cents -= 1
        count += 1
        pennies += 1
        
# show no. of cions and add new lines for classification of cions
print(f"no. of cions= {count}")

# add new lines for each classification of cions
print(f"quarters = {quarters}, dimes = {dimes}, nickels = {nickels}, pennies = {pennies}")